### PR TITLE
Applying update to generators

### PR DESCRIPTION
**Pull request by PhillypHenning**
Because of the dynamic name generation different length identifiers need to be created to created. For example, a continuously used state bucket is the `X-tf` named bucket, if a `X-logs` bucket needs to be created as well, then a separate identifier needs to be created for it. 

This merges all common identifier generations into a single file to clean up repeat files.

### CODE REVIEW

**Review by LeoDiazL — APPROVED** [review]
Looking good

File: operations/_scripts/generate/generate_buckets_identifiers.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -e
+
+GITHUB_IDENTIFIER="$(echo $($GITHUB_ACTION_PATH/operations/_scripts/generate/generate_identifier.sh) | tr '[:upper:]' '[:lower:]')"
+
+function generateIdentifiers () {
```
Comment (armab) [inline operations/_scripts/generate/generate_buckets_identifiers.sh:7]: Same as for the other function, naming and description would be nice to haves.

And quickly looking at it, I didn't understand what kind of Identifiers the function generates.
`generate_bucket_names()` or `generate_bucket_identifiers` perhaps would be more clear?

**Review by armab — CHANGES_REQUESTED** [review]
A brief description for the PR would be nice to have a better understanding as I had to spend a few cycles to understand the high-level idea first.
I understand some refactoring is happening and left a couple of comments.

**Comment by PhillypHenning** [comment]
Gotcha - Sorry about that @armab, this was work that was ported over from an upgrade to the docker-to-ec2. (Hence why I was pushing for a commons last week)

I'll post up some cleanup shortly

**Review by armab — APPROVED** [review]
Thanks!

**Pull request merged**